snowexsql/api.py

```diff
@@ -218,7 +255,9 @@ def all_instruments(self):
         Return all distinct instruments in the data
         """
         with db_session(self.DB_NAME) as (session, engine):
-            qry = session.query(self.MODEL.instrument).distinct()
+            qry = session.query(Instrument.name).join(
+                self.MODEL, Instrument.id == self.MODEL.instrument_id
+            ).distinct()
```

Comment:
Same thought as with Observers - No join needed from my perspective. There should only be unique instrument entries in the table.

Reply:
The only upside to a join would be to limit these to observers in the specific table (points vs layers). But querying on an observer that's not there will just return no points, so it doesn't really matter if we include extra observers